FBC/FunctionalEntities/BusinessRegistries.rdf

```diff
@@ -593,8 +587,9 @@ NAICS was developed under the auspices of the Office of Management and Budget (O
 	<owl:Class rdf:about="&fibo-fbc-fct-breg;RegistrationAddress">
 		<rdfs:subClassOf rdf:resource="&fibo-fnd-plc-adr;PhysicalAddress"/>
 		<rdfs:label>registration address</rdfs:label>
-		<skos:definition>an address for an organization used to register that organization in any registry, such as its legal address, headquarters address, a local address used to obtain a license or pay taxes in some jurisdiction, etc.</skos:definition>
+		<skos:definition>address for an organization used to register that organization in any registry, such as its legal address, headquarters address, a local address used to obtain a license or pay taxes in some jurisdiction, etc.</skos:definition>
```

Comment:
but later we say that hasRegistrationAddress can apply to people.

Reply:
fixed

Reply:
OK, but this still leaves an unclear distinction between RegisteredAddress and RegistrationAddress as per #236. Seems like RegistrationAddress is specific to being in a Registry as opposed to some other official record? Why then does RegistrationAddress not subclass RegisteredAddress? And do we even need separate classes - why not represent the distinction via the 2 properties (one subProperty of the other) with range of PhysicalAddress? If a previously recorded (registered) address is suddenly included in a registry then does it make sense for its class to change? It's still the same address!

Reply:
I think the distinction is much more nuanced than it was originally - will investigate eliminating registration address and possibly broadening the definition of registered address.